src/components/CommentsAndLikes.tsx

```diff
@@ -94,14 +86,36 @@ const CommentsAndLikes = ({
   };
 
   useEffect(() => {
-    const getCommAndLikesCount = async () => {
-      let data = await getPostCommentsCount(postId);
-      dispatch(postActions.setPostCommentDeleted(false));
-      dispatch(postActions.setPostCommented(false));
-      setCount(data);
+    const getPostCommentsCount = async () => {
+      try {
+        const response = await axios.get(
+          `http://localhost:7000/api/posts/getPostCommentsCount/${postId}`
+        );
+
+        setCommentsCount(response.data.comment_count);
+      } catch (err) {
+        console.log(err);
+      }
+    };
+
+    getPostCommentsCount();
+  }, [postComments]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    const getPostLikesCount = async () => {
+      try {
+        const response = await axios.get(
+          `http://localhost:7000/api/posts/getPostLikesCount/${postId}`
```

Comment:
We should refactor this to usa base url so it's easier to change latter on and not so long.